### PR TITLE
Add draft for manuscript template

### PR DESCRIPTION
**Pull request by klngwll**
Add template for Instance type Manuscript.

### CODE REVIEW

File: vue-client/src/resources/json/combinedTemplates.json [hunk 1]
```diff
@@ -964,6 +964,200 @@
...
+                "familyName": "",
+                "givenName": "",
+                "lifeSpan": ""
+              },
+              {
+                "@type": "ComplexSubject",
+                "inScheme": {
+                  "@id": "https://id.kb.se/term/sao"
+                },
+                "termComponentList": [
```
Comment (niklasl) [inline vue-client/src/resources/json/combinedTemplates.json:1135]: Untimely perhaps, but given that we're working to get rid of at least complex subjects with geographic terms, should we "discourage" them now but not putting them in templates?

File: vue-client/src/resources/json/combinedTemplates.json [hunk 2]
```diff
@@ -964,6 +964,200 @@
...
+                    "@id": "https://id.kb.se/relator/author"
+                  }
+                ]
+              },
+              {
+                "@type": "Contribution",
+                "agent": null,
+                "role": [
+                  {
+                    "@id": "https://id.kb.se/relator/unspecifiedContributor"
```
Comment (niklasl) [inline vue-client/src/resources/json/combinedTemplates.json:1107]: Pointless?

File: vue-client/src/resources/json/combinedTemplates.json (hunk 1 above)
Comment (klngwll) [reply vue-client/src/resources/json/combinedTemplates.json:1135]: I know! If we can land that before or very close to the next release it we should ofc remove it, otherwise it is a big time-saver (unfortunately), my reasoning now is to maybe keep it until we can be more sure.

File: vue-client/src/resources/json/combinedTemplates.json [hunk 3]
```diff
@@ -964,6 +964,200 @@
...
+                  "@type": "Place",
+                  "label": [
+                    ""
+                  ]
+                }
+              ]
+            }
+          ],
+          "carrierType": [{"@id": "https://id.kb.se/term/rda/Volume"}],
+          "baseMaterial": [{"@id": "https://id.kb.se/term/rda/material/Paper"}],
```
[the remark this answers is not in the document]
Comment (klngwll) [reply vue-client/src/resources/json/combinedTemplates.json:1039]: Good catch!

File: vue-client/src/resources/json/combinedTemplates.json (hunk 2 above)
Comment (klngwll) [reply vue-client/src/resources/json/combinedTemplates.json:1107]: Hmm maybe, I need to investigate what the actual use-case would be to specify unspecified. My suspicion is that it is used when there is only one contributor and it is not the author (which usually has been inferred when missing).

Comment (klngwll) [reply vue-client/src/resources/json/combinedTemplates.json]: There was nothing in the instructions regarding this, so I removed it.

File: vue-client/src/resources/json/combinedTemplates.json (hunk 1 above)
Comment (niklasl) [reply vue-client/src/resources/json/combinedTemplates.json:1131]: I agree. Would you like to create a  issue/task in the Geo story for removing complex-geo-subjects for _all_ templates?

Comment (klngwll) [reply vue-client/src/resources/json/combinedTemplates.json:1131]: Subtask created ✅

File: vue-client/src/resources/json/combinedTemplates.json [hunk 4]
```diff
@@ -964,6 +964,196 @@
...
+            "@id": "https://id.kb.se/TEMPID#it"
+          },
+          "bibliography": [],
+          "descriptionConventions": [
+            {
+              "@id": "https://id.kb.se/marc/Isbd"
+            },
+            {
+              "@type": "DescriptionConventions",
+              "code": "rda"
```
Comment (niklasl) [inline vue-client/src/resources/json/combinedTemplates.json:984]: A link would be good (as discussed); but that is reasonably done in a separate PR for all templates.

Comment (klngwll) [reply vue-client/src/resources/json/combinedTemplates.json:984]: Yes, let's do that separate.

File: vue-client/src/resources/json/combinedTemplates.json [hunk 5]
```diff
@@ -964,6 +964,196 @@
...
+                    "@id": "https://id.kb.se/relator/author"
+                  }
+                ]
+              },
+              {
+                "@type": "Contribution",
+                "agent": null,
+                "role": [
+                  {
+                    "@id": "https://id.kb.se/relator/author"
```
[the remark this answers is not in the document]
Comment (klngwll) [reply vue-client/src/resources/json/combinedTemplates.json:1098]: Maybe it should be https://id.kb.se/relator/dubiousAuthor ? 😉 
It is not uncommon (lots of aggregated writings I suppose) but it is not more common than scribe, translator etc from what I can see. Let's remove and take the discussion if it pops up. Having too many prefilled values is a always risky.

[the remark this answers is not in the document]
Comment (niklasl) [reply vue-client/src/resources/json/combinedTemplates.json]: Sounds good! Yes, we reasonably lack more powerful editing options for adding, especially linking, things!

**Review by niklasl — APPROVED** [review]
LGTM!
